src/link/MachO/Trie.zig

```diff
@@ -0,0 +1,436 @@
+/// Represents export trie used in MachO executables and dynamic libraries.
```

Comment:
We have top level doc comments:

```suggestion
//! Represents export trie used in MachO executables and dynamic libraries.
```